auth/auth0/test/auth0Authentificaction.test.ts

```diff
@@ -0,0 +1,111 @@
+import 'jest'
```

Comment:
Remove your own Auth0 values before committing. Can you make this easy to configure?

Reply:
All is mocked here (all requests are catch and mock by `nock` to avoid internet dependency), I have started with a real jwt but it wasn't a viable solution (jwt have expiration date…).

So yes, I can change the value of `AUTH0_DOMAIN` and `AUTH0_AUDIENCE` to something more generic 😉